ci_tools/json_pytest_output.py

```diff
@@ -61,7 +61,7 @@ def     mini_md_summary(title, outcome, failed_tests):
     summary = ""
 
     failed_pattern = re.compile(r".*FAILED.*")
-    languages = ('c', 'fortran', 'python')
+    languages = ('c', 'fortran', 'python', 'cuda')
```

Comment:
Have you tested this? Does it not need C, Fortran and Python to run to make sense?

Reply:
this is a search and replace mistake.
but after reading the file my understanding that this file generate the summary of the tests shouldn't we generate the `cuda` summary too?

Reply:
Yes, but I'm not sure if the file handles each language individually or in order. I haven't checked though

Reply:
If you want to use the script you should add a step to the `pytest_run_cuda` action similar to this:
https://github.com/pyccel/pyccel-cuda/blob/devel/.github/actions/pytest_run/action.yml#L72-L83

Reply:
fixed in [e2e51c4](https://github.com/pyccel/pyccel-cuda/pull/32/commits/e2e51c4ca95a5245c578d232895e3e26a1708377)